content/end.html

```diff
@@ -70,51 +75,62 @@ <h3> Inspirations </h3>
 <p class="full">
 Systems Design Engineering 2017 Class Profile
```

Comment:
Maybe we should provide the latest SYDE profile which is still online instead? https://ndey96.github.io/syde_2019_class_profile.pdf